data/themes.yml

```diff
@@ -136,7 +136,7 @@
 - { colors: '#212121,#212121,#3F51B5,#FFFFFF,#3F51B5,#FFFFFF,#3F51B5,#3F51B5', name: 'Material Dark Indigo' }
 - { colors: '#212121,#212121,#607D8B,#FFFFFF,#607D8B,#FFFFFF,#607D8B,#607D8B', name: 'Material Dark Blue Grey' }
 - { colors: '#2F343B,#2F343B,#FD7532,#FFFFFF,#FD7532,#FFFFFF,#FD7532,#FD7532', name: 'Hummingbird' }
-- { colors: '#CEC18A,#ffffff,#000000,#FFFFFF,#FFFFFF,#000000,#ffff00,#000000', name: 'Pittsburgh Penguins' }
+- { colors: '#CEC18A,#ffffff,#000000,#FFFFFF,#FFFFFF,#000000,#ffff00,#000000', name: 'Pittsburgh Penguins (old)' }
```

Comment:
Please don't use `(old)` or `(new)` in theme names, if a theme's colors need updating, please update the colors and state the reason why in the PR description.

Btw, I don't even see the `(new)` theme in this PR.

Reply:
The Pittsburgh Penguins have a new color scheme so I didn't want to get rid of the old version but also wanted to add the new one. I don't know why the new one got missed in this PR so I can add it too, but I can add it and fix the descriptions.